rero_ils/modules/utils.py

```diff
@@ -1093,3 +1104,10 @@ def get_objects(record_class, query):
     """
     for hit in query.source().scan():
         yield record_class.get_record_by_id(hit.meta.id)
+
+
+def truncate_string(str_input, max_length, ellipsis='...'):
```

Comment:
Seems not used.

Reply:
use here : https://github.com/zannkukai/rero-ils/commit/5e4c5d607e93af46c1a66eaaab6e9a542b58bd38#diff-a00f4dffd67a253e14c92920179ecd0e39e3c57346a77bcb47c29a1437c8dccfR45

I rename it because line are limite to 80 characters.